pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -319,6 +319,22 @@ func mapNodeNameToServerName(nodeName types.NodeName) string {
 	return string(nodeName)
 }
 
+// getNodeNameByID maps instanceid to types.NodeName
+func (os *OpenStack) GetNodeNameByID(instanceID string) (types.NodeName, error) {
+	client, err := os.NewComputeV2()
+	var nodeName types.NodeName
+	if err != nil {
+		return nodeName, err
+	}
+
+	server, serr := servers.Get(client, instanceID).Extract()
```

Comment:
For go style, please use err.

Reply:
I think it is fine. Go's variable shadowing rules can be awkward sometimes. @zetaab is being explicit about variable names - it doesn't seem like a big deal.